app/frontend/src/signOutAfterInactivity.js

```diff
@@ -0,0 +1,28 @@
+var timeoutInMinutes = 30
+
+var timeoutId; 
+
+function resetTimer() { 
+    window.clearTimeout(timeoutId)
+    startTimer();
+}
+
+function clickSignOut() {
+    document.getElementById('sign_out').click();
```

Comment:
Following on from @cesidio observation, I'd advise against simulating user interaction. Instead find where the session information is stored and delete that.  It is most-likely stored in a cookie, but it might also be in Local or Session storage.

Reply:
i agree, yes good point @cesidio